agent/agent-profiler/docs/README.md

```diff
@@ -70,8 +70,25 @@ Profiler may also be enabled via adding a config section to applicationinsights.
     }
 ```
 
-The profiler periodically polls for configuration changes made within the UI. This can be adjusted via
-`configPollPeriod`.
+### Json Configuration Parameters
+`configPollPeriodSeconds` - The profiler periodically polls for configuration changes made within the UI.
+
+`removeEnvironmentData` - If set to true (and no custom profile are provided), memory and CPU profiling will 
+use a profile similar to the default `profile` configuration with the following settings disabled:
+- jdk.JVMInformation
+- jdk.InitialSystemProperty
+- jdk.OSInformation
+- jdk.InitialEnvironmentVariable
+- jdk.SystemProcess
+
+`customMemoryProfile` - Path to a custom jfc configuration file. This configuration will be used in the event of a
```

Comment:
what do you think of renaming to `memoryTriggeredSettings` (and `cpuTriggeredSettings`)?

"settings" because it matches the JFR command-line option, and avoids multiple meanings of "profile"

and also, what do you think about making it have default value `profile` (corresponding to the JFR option), and supporting another non-file value `profile-without-env-data` (or similar), and getting rid of `removeEnvironmentData`?

Reply:
sounds good